src/pages/docs/ai-transport/features/sessions-identity/online-status.mdx

```diff
@@ -0,0 +1,279 @@
+Clients require the [`presence`](/docs/auth/capabilities) capability on the channel to leave presence.
+</Aside>
+
+<Code>
+```javascript
+// Client code
+import * as Ably from "ably";
+
+const ably = new Ably.Realtime({
+  authUrl: "/api/auth/token"
+});
+
+const channel = ably.channels.get("{{RANDOM_CHANNEL_NAME}}");
+
+// Leave presence when the user marks themselves offline
+await channel.presence.leave();
+```
+</Code>
+
+<Code>
+```javascript
+// Agent code
+import * as Ably from "ably";
+
+const ably = new Ably.Realtime({
+  key: "{{API_KEY}}",
+  clientId: "weather-agent"
+});
+
+const channel = ably.channels.get("{{RANDOM_CHANNEL_NAME}}");
+
+// Leave presence when the agent shuts down
+await channel.presence.leave();
+```
+</Code>
+
+### Going offline after disconnection <a id="disconnection"/>
+
+When a client loses connection unexpectedly, Ably detects the lost connection and automatically leaves the client from the presence set.
+
+By default, clients remain present for 15 seconds after an abrupt disconnection. This prevents excessive enter/leave events during brief network interruptions. If the client reconnects within this window, they remain in the presence set without triggering leave events.
+
+Use the `transportParams` [client option](/docs/api/realtime-sdk#client-options) to configure disconnection detection and presence lifecycle behaviour. After an abrupt disconnection, the `heartbeatInterval` transport parameter controls how quickly Ably detects the dead connection, while the `remainPresentFor` option controls how long the member is kept in presence before Ably emits the leave event.
+
+<Aside data-type="note">
+For more information, see [Handle unstable connections and failures](https://ably.com/docs/presence-occupancy/presence?lang=javascript#unstable-connections).
+</Aside>
+
+For example, if implementing resumable agents using techniques like durable execution, configure a longer `remainPresentFor` period to allow time for the new agent instance to come online and resume processing before the previous instance appears as offline. This provides a seamless handoff:
+
+<Code>
+```javascript
+// Agent code
+import * as Ably from "ably";
+
+const ably = new Ably.Realtime({
+  key: "{{API_KEY}}",
+  clientId: "weather-agent",
+  // Allow 30 seconds for agent resume and reconnection
+  transportParams: {
+    remainPresentFor: 30000
+  }
+});
+```
+</Code>
+
+## Viewing who is online <a id="viewing-presence"/>
+
+Participants in a session can query the current presence set or subscribe to presence events to see who else is online and react to changes in realtime. Users might want to see which agents are processing work, while agents might want to detect when specific users are offline to pause or cancel work.
+
+<Aside data-type="note">
+Clients require the `subscribe` or `presence-subscribe` [capability](/docs/auth/capabilities) on the channel to retrieve the presence set and subscribe to presence events.
```

Comment:
```suggestion
Clients require the `subscribe` [capability](/docs/auth/capabilities) on the channel to retrieve the presence set and subscribe to presence events.
```
(There is no `presence-subscribe` permission, only a `presence-subscribe` mode.)